x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/create_source_editor.js

```diff
@@ -46,19 +46,17 @@ export class CreateSourceEditor extends Component {
 
   render() {
 
-    if (this.state.url === null) {
-      return null;
-    }
```

Comment:
this was the actual bug, that it didn't show the hint.

Reply:
This is still needed, but instead of returning null, it should return a loading config message. 

With out this, `helpText={this.state.url ? null : NO_TILEMAP_LAYER_MSG}` will display `NO_TILEMAP_LAYER_MSG` until `_loadUrl` completes. So users with a config could see `NO_TILEMAP_LAYER_MSG` if there is a slow connection between the client and Kibana

Reply:
yes you're right